.github/workflows/deploy-happy-stack.yml

```diff
@@ -168,6 +168,8 @@ jobs:
     timeout-minutes: 30
     name: e2e-tests ${{ matrix.project }} ${{ matrix.shardCurrent }} of ${{ matrix.shardTotal }}
     runs-on: ubuntu-22.04
+    needs:
+      - upgrade
```

Comment:
Thanks for catching this! Can we add the same `need` to `Run E2E Logged In tests` too? Thank you!